packages/maskbook/src/plugins/VCent/define.tsx

```diff
@@ -15,6 +15,7 @@ export const VCentPluginDefine: PluginConfig = {
     postInspector: function Component(): JSX.Element | null {
         const tweetAddress = usePostInfoDetails('postID')
         if (!tweetAddress) return null
+        if (window.location.href.includes('v.cent.co')) return null
```

Comment:
Hi @rob-lw ! The goal of this change was that we aren't loading the component for every tweet in the feed, only the tweet that has been clicked on. Could we update to this?

`if (window.location.pathname.split('/')[3] != tweetAddress) return null`

Reply:
Hi @chejazi. ah okay that is a misunderstanding on my part. I have updated to this.